src/orquestra/sdk/_ray/_build_workflow.py

```diff
@@ -214,7 +214,14 @@ def _ray_remote(*inner_args, **inner_kwargs):
                     deserialize=serialization,
                 )
 
-                wrapped_return = wrapped(*inner_args, **inner_kwargs)
+                try:
+                    wrapped_return = wrapped(*inner_args, **inner_kwargs)
+                except SystemError as e:
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Where does `SystemError` come from? Let's document what raises this exception.